charts/ccd-case-print-service/values.preview.template.yaml

```diff
@@ -0,0 +1,11 @@
+nodejs:
+  releaseNameOverride: ${SERVICE_NAME}-nodejs
+  ingressHost: ${SERVICE_FQDN}
+  image: ${IMAGE_NAME}
+  environment:
+    IDAM_PRINT_SERVICE_KEY: ${IDAM_SERVICE_KEY}
+    IDAM_API_URL: http://betaDevBccidamAppLB.reform.hmcts.net
```

Comment:
I think this is an old IDAM endpoint. Now we have IDAM on AAT, we can point to that one. Same way as we are doing with all the other services. Please compare with: 
https://github.com/hmcts/ccd-api-gateway/blob/daf9476b8bc63c3b905c000c942cb1bd7a79d359/charts/ccd-api-gateway-web/values.template.yaml#L17

Reply:
Modified now